robot_sort/test_robot.py

```diff
@@ -36,10 +36,11 @@ def test_sorting_random_list(self):
         robot.sort()
         self.assertEqual(robot._list, sorted(self.random_list))
 
-    # def test_stretch_times(self):
-    #     robot = SortingRobot(self.small_list)
-    #     robot.sort()
-    #     self.assertLess(robot._time, 110)
+    def test_stretch_times(self):
+    # my code only passes this test - can I get partial credit? ;-)
```

Comment:
even when your stretch only passes 1 test, you already granted yourself 90 points, which is enough for a 3, great job on your SC and the entire week in general. Definitely looking forward to seeing what you can do with data Structures next week.

Reply:
Thanks, Jorge. Enjoy working with you.